backend/src/controllers/reports/helpers/index.js

```diff
@@ -28,26 +28,40 @@ const loadTemplate = async (path, encoding = "base64") => {
   return data;
 };
 
+/**
+ * Generates the API body for creating a document using the provided data and template.
+ *
+ * @param   {object}        data                  - The data to be used in the document.
+ * @param   {string}        templateFileName      - The name of the template file to use.
+ * @param   {string}        [templateType="docx"] - The type of the template file (default: "docx").
+ * @param   {string}        [reportName="report"] - The name of the report (default: "report").
+ * @param   {string}        [convertTo="pdf"]     - The format to convert the document to (default: "pdf").
+ * @returns {object | null}                       Returns an object containing the API body for creating a document or null if the input is invalid.
+ */
 const getDocumentApiBody = async (
   data,
   templateFileName,
   templateType = "docx",
   reportName = "report",
   convertTo = "pdf"
```

Comment:
if convertTo and reportName are not being used, should we remove it from the params that we are passing the function

Reply:
I'd like to do that, but I worry that other controllers might be passing those parameters.

but, I'll find out exacly which controllers use getDocumentApiBody, and if none of them use those parameters, I'll remove them.